File: src/main/java/models/City.java
package models;

import java.util.ArrayList;
import java.util.List;

public class City {
    private String name;
    private String country;
    private List<Station> bikeStations = new ArrayList<Station>();

    public void addStation(Station bikeStation) {
        this.bikeStations.add(bikeStation);
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public List<Station> getStations() {
        return bikeStations;
    }

    public void setStations(List<Station> bikeStations) {
        this.bikeStations = bikeStations;
    }
}

File: src/main/java/models/city.py
class City:
    def __init__(self, name="", country="", bikeStations=[]):
        self._name = name
        self._country = country
        self._bikeStations = bikeStations

    def addStation(self, bikeStation):
        self._bikeStations.append(bikeStation)

    def getName(self):
        return self._name

    def setName(selfname):
        self._name = name

    def getCountry(self):
        return self._country
    
    def setCountry(selfcountry):
        self._country = country
    
    def getStations(self):
        return self._bikeStations
    
    def setStations(self, bikeStations):
        self._bikeStations = bikeStations
